Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    zendesk_apps_tools (2.12.1)
+    zendesk_apps_tools (2.12.2)
```

Comment:
Waiting for [this](https://github.com/zendesk/zendesk_apps_tools/pull/267) to be approved and merged. Will remove from this commit.